backend/stories.json

```
@@ -13,6 +13,7 @@
             "Py py loved taking inspiration from the books they read and dreamed up ideas of inventions that would change the world.",
             "Py Py was telling other pythons everything that it learned from the books and encouraged others to read as well.",
             "Py Py decided to publish its short stories"
```

would probably need to add a comma (`,`) here so that there aren't errors when running the app

```suggestion
            "Py Py decided to publish its short stories",
```